src/main/cli/cli.c

```diff
@@ -4820,8 +4820,8 @@ if (buildKey) {
                 cliPrint("configured");
             }
         }
+        cliPrint(", version = ");
```

Comment:
Or be brave ;-)
```suggestion
        cliPrintf(", version = %s", gpsData.platformVersion != UBX_VERSION_UNDEF ? ubloxVersionMap[gpsData.platformVersion].str : "unknown");
```

Reply:
Yeah was thinking about to remove the else